Tasks/Maven/maventask.ts

```diff
@@ -220,29 +221,13 @@ function configureMavenOpts() {
 
 // Publishes JUnit test results from files matching the specified pattern.
 function publishJUnitTestResults(testResultsFiles: string) {
-    var matchingJUnitResultFiles: string[] = undefined;
-
-    // Check for pattern in testResultsFiles
-    if (testResultsFiles.indexOf('*') >= 0 || testResultsFiles.indexOf('?') >= 0) {
-        tl.debug('Pattern found in testResultsFiles parameter');
-        var buildFolder = tl.getVariable('System.DefaultWorkingDirectory');
-        tl.debug(`buildFolder=${buildFolder}`);
-        var allFiles = tl.find(buildFolder);
-        matchingJUnitResultFiles = tl.match(allFiles, testResultsFiles, {
-            matchBase: true
-        });
-    }
-    else {
-        tl.debug('No pattern found in testResultsFiles parameter');
-        matchingJUnitResultFiles = [testResultsFiles];
-    }
-
+    let matchingJUnitResultFiles: string[] = ffl.findFiles(testResultsFiles, false, tl.getVariable('System.DefaultWorkingDirectory'));
```

Comment:
Don't do this. This is a completely different globbing library and is going to break customers.